### PR TITLE
popup window

### PR DESCRIPTION
**Pull request by mumbereh**
### **Description**
In this milestone of my  portfolio website, I have  implement a popup window that includes the project details in both desktop and mobile.

### **I implemented the following in this branch**

> 1. I JavaScript array to store all of the information for all projects.
> 2. I stored the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to source.
> 3. I  updated the main page so that the projects section is created dynamically using the information stored in that JavaScript object. And all of the HTML in that section is created when the page loads.
> 4. Finally I implemented  popup window.
> 5. When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
> 6. When the user clicks (or taps) the close (X) button, the popup disappears.
> 7. In order to associate each project with the popup window details,
> 8.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -51,120 +52,158 @@ <h2>Let's Connect</h2>
...
+<            <div id="portfolios"></div>
+</section>
+
+<section class="container">
+    <div class="pro-container"></div>
+    
+     <!--<div class="card" id="01">
+        <div id="first">
+            <img src="images/61.png" alt="">
         </div>
```
Comment (LionRouge1) [inline index.html]: - [ ] Please do not push commented out code to your code base, It make your project look messy. Therefore I suggest you remove all commented out code from your project.

File: index.html [hunk 2]
```diff
@@ -51,120 +52,158 @@ <h2>Let's Connect</h2>
...
+            <li class="other-skills">css</li>
+            <li class="other-skills">javascript</li>
+        </ul>
+        <button id="click" class="card-btn" onclick="openPopup()">See Project</button>
+    </div>
+</div> 
+    <div class="card" id="02">
+        <div class="pic">
+            <img  src="images/4.png" alt="">
         </div>
```
Comment (LionRouge1) [inline index.html]: - [ ] Same here as pointed out earlier :point_up_2:

File: index.html [hunk 3]
```diff
@@ -51,120 +52,158 @@ <h2>Let's Connect</h2>
...
+            <li class="ruby">Ruby on rails</li>
+            <li class="other-skills">css</li>
+            <li class="other-skills">javascript</li>
+        </ul>
+        <button id="click" class="card-btn" onclick="openPopup()">See Project</button>
+    </div>
+</div>  -->
+
+
+
```
Comment (LionRouge1) [inline index.html]: - [ ] Same here as pointed out earlier :point_up_2:

File: index.html [hunk 4]
```diff
@@ -51,120 +52,158 @@ <h2>Let's Connect</h2>
...
+</a>
+<a href="https://github.com/mumbereh/my-portfolio.git">
+<button class="link-btn" >See source<img class="search" src="pop-images/popup.png"></button>
+</a>
+<p id="close-btn">&times;</p> 
+</div> 
+
+
+ 
+           <--</section> -->
```
Comment (LionRouge1) [inline index.html]: - [ ] Same here as pointed out earlier :point_up_2:

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi **Team**, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good job implementing the mobile menu, it looks almost identical to the figma design :+1: 
- Nice work following JavaScript best practices :1st_place_medal:  
- All linters passed :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

[OPTIONAL]
- [ ] Nice work implementing the mobile menu, everything looks fine :+1: :sunglasses: However it would be nice to add `position: fixed` to the `nav` in your `style.css` file at line 132 to prevent it scroll with the page.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by 1GORDON — COMMENTED** [review]
##
## Hi @mumbereh, and @Wineshuga ✋

Good job so far!

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

### This project will be marked as Invalid and won't affect the number of your weekly reviews.

- [ ] You have submitted a project with failing linter checks. Kindly make sure your linter checks are passing and are in green.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mumbereh** [comment]
Hello there I thank you for receiving my request for review for PR:
https://github.com/mumbereh/my-portfolio/pull/13.

The first reviewer requested changes. I worked on them and submitted them
for the second review.

*The second reviewer *
On Tue, 20 Jun 2023 at 08:09, Gordon Tinyefuza ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
> Hi @mumbereh <https://github.com/mumbereh>, and @Wineshuga
> <https://github.com/Wineshuga> ✋
>
> Good job so far!
>
> Hello ✋, you have done well reaching this point of the milestone 🤝 . But
> there are some issues that you still need to work on to go to the next
> project but you are almost there!
> This project will be marked as Invalid and won't affect the number of your
> weekly reviews.
>
>    -  You have submitted a project with failing linter checks. Kindly
>    make sure your linter checks are passing and are in green.
>
> Cheers, and Happy coding!👏👏👏
>
> Feel free to leave any questions or comments in the PR thread and tag me
> @1GORDON if something is not 100% clear.
>
> *Please, do not open a new Pull Request for re-reviews. You should use the
> same Pull Request submitted for the first review, either valid or invalid
> unless it is requested otherwise.*
>

I did my best basing on the information below
https://microverse-students.slack.com/archives/C47D4N35M/p1686654751539179
Marco Lizza
Hey Micronauts ***@***.****! [image: :wave:]I hope you're doing well! We are
aware that most of you are experiencing Webhint issues, we are currently
working on a solution[image: :male-technologist:].For now, please go ahead
and request for review if you only have the Webhint error on your PR. Our
CRs would be happy to review your PRs [image: :slightly_smiling_face:].
Show more

[image: review.PNG]
kindly advise me. since the linter error seems not to be mine but
technical.

> Thank you i really appreciate all. Message ID:
> ***@***.***>
>

**Review by Shisui6 — APPROVED** [review]
Hi @mumbereh, 

# Approved 🪄
Your project is complete! There is nothing else to say other than... it's time to merge it 💎
Congratulations! 🎉.  

## Highlights 🎯
- Descriptive PR title and summary ✅
- Follows Github flow ✅
- Good commit messages ✅

## Optional suggestions 🪶

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by wayungi — COMMENTED** [review]
Hi @mumbereh , 
image

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ ❌ 

- [ ] You have submitted a PR that was is already approved about 10 minutes ago. Please submit a new PR for every milestone

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by V-Blaze — APPROVED** [review]
Hi @mumbereh  , 

# Approved 🟢 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy

## Highlights

- Correct GitHub Flow💯

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mumbereh** [comment]
Thank you

**Pull request merged**